Source/astcenc_pick_best_endpoint_format.cpp

```diff
@@ -968,6 +971,33 @@ void determine_optimal_set_of_endpoint_formats_to_use(
 				best_error_index = j;
 			}
 		}
+#else
+		// find best mode, SIMD N-wide way
+		vint vbest_error_index(-1);
+		vfloat vbest_ep_error(1e30f);
+		vint lane_ids = vint::lane_id();
+		for (int j = 0; j < MAX_WEIGHT_MODES; j += ASTCENC_SIMD_WIDTH)
```

Comment:
It should always be the case, but can we add an assert that MAX_WEIGHT_MODES is an exact multiple of SIMD_WIDTH, just in case we optimize the weight modes used later.